app/styles/_utilities.css

```
@@ -94,3 +93,48 @@
     animation: none;
   }
 }
+
+@layer components {
+  .embla__slide {
+    @apply transition-all duration-500 ease-out;
+    transform-origin: bottom center; /* 회전축을 아래로 */
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)